modules/file-integrity-node-status-failure.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/file-integrity-operator-understanding.adoc
+
+[id="file-integrity-node-status-failure_{context}"]
+=  FileIntegrityNodeStatus failure status example
+
+To simulate a failure condition, modify one of the files AIDE tracks. For
+example, modify `/etc/resolv.conf` on one of the worker nodes:
+
+[source,terminal]
+----
+$ oc debug node/ip-10-0-130-192.ec2.internal
+----
+
+.Example output
+[source,terminal]
+----
+Creating debug namespace/openshift-debug-node-ldfbj ...
+Starting pod/ip-10-0-130-192ec2internal-debug ...
+To use host binaries, run `chroot /host`
+Pod IP: 10.0.130.192
+If you don't see a command prompt, try pressing enter.
+sh-4.2# echo "# integrity test" >> /host/etc/resolv.conf
+sh-4.2# exit
+
+Removing debug pod ...
+Removing debug namespace/openshift-debug-node-ldfbj ...
+----
+
+After some time, the `Failed` condition was reported in the results array of the
+corresponding `FileIntegrityNodeStatus`. The previous `Succeeded` condition is
+retained, which allows you to pinpoint the time the check failed.
+
+[source,terminal]
+----
+$ oc get fileintegritynodestatuses.fileintegrity.openshift.io/worker-fileintegrity-ip-10-0-130-192.ec2.internal -ojsonpath='{.results}' | jq -r
```

Comment:
I tried locally, with "jq" it failed:
$ oc get fileintegritynodestatuses.fileintegrity.openshift.io -ojsonpath='{.items[*].results}'  | jq
parse error: Invalid numeric literal at line 1, column 5
$ oc get fileintegritynodestatuses.fileintegrity.openshift.io -ojsonpath='{.items[*].results}'
[map[condition:Succeeded lastProbeTime:2020-10-09T08:01:08Z] map[condition:Errored errorMessage:Error initializing the AIDE DB: Invalid configureline error lastProbeTime:2020-10-09T08:05:07Z]] [map[condition:Errored errorMessage:Error initializing the AIDE DB: Invalid configureline error lastProbeTime:2020-10-09T08:05:07Z]] [map[condition:Errored errorMessage:Error initializing the AIDE DB: Invalid configureline error lastProbeTime:2020-10-09T08:05:07Z]] [map[condition:Errored errorMessage:Error initializing the AIDE DB: Invalid configureline error lastProbeTime:2020-10-09T08:05:07Z]] [map[condition:Errored errorMessage:Error initializing the AIDE DB: Invalid configureline error lastProbeTime:2020-10-09T08:05:07Z]] [map[condition:Succeeded lastProbeTime:2020-10-09T08:00:57Z] map[condition:Errored errorMessage:Error initializing the AIDE DB: Invalid configureline error lastProbeTime:2020-10-09T08:05:08Z]]

Reply:
you need `jq -r`

Reply:
from http://file.rdu.redhat.com/~ahardin/Oct-7-2020/file-integrity-operator/security/file_integrity_operator/file-integrity-operator-understanding.html, it is jq; besides, I met the same error for "jq -r".
@pdhamdhe Could you help to try and give feedback? Thanks.

Reply:
@xiaojiey I checked and the command works fine. we should use the below command if we are not mentioning the object name.
`$ oc get fileintegritynodestatuses.fileintegrity.openshift.io -ojsonpath='{.items[*].results}' | jq`
OR
`oc get fileintegritynodestatuses.fileintegrity.openshift.io/worker-fileintegrity-ip-10-0-213-221.us-east-2.compute.internal -ojsonpath='{.results}' | jq
`

Reply:
Thanks. Then it should be a env issue.

Reply:
Thank you for confirming!